### PR TITLE
[PORT][8.0] vertical-ngo/transport_plan becomes stock_shipment_management

### PR DESCRIPTION
**Pull request by yvaucher**
This is a refactoring of module transport_plan from vertical-ngo

It allows to manage a shipment from departure to arrival.

Depends on:
- [x] #51 

TODO:
- [x] Add translations

### CODE REVIEW

File: stock_shipment_management/model/stock_move.py [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+                if ship.state == 'confirmed':
+                    ship.signal_workflow('transit_start')
+            for ship in self.mapped('arrival_shipment_id'):
+                if ship.state == 'in_transit':
+                    ship.signal_workflow('transit_end')
+        return res
+
+    @api.multi
+    def _picking_assign(self, procurement_group, location_from, location_to):
+        """ When an arrival transit move is splitted, we create a backorder
```
Comment (lepistone) [inline stock_shipment_management/model/stock_move.py]: split

**Comment by yvaucher** [comment]
@lepistone Amended 2 typos of split

File: stock_shipment_management/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+#    GNU Affero General Public License for more description.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+{"name": "Shipment Management",
+ "version": "0.1",
+ "author": "Camptocamp",
+ "category": "Transportation",
```
Comment (lepistone) [inline stock_shipment_management/__openerp__.py:24]: Maybe we could use an existing category, like 'Warehouse Management'?

**Comment by lepistone** [comment]
Thanks, :+1:

File: stock_shipment_management/__openerp__.py (hunk 2 above)
Comment (yvaucher) [inline stock_shipment_management/__openerp__.py:24]: @lepistone This module is not there for warehouse management as we want to know when a shipment is arrived. Does a warehouse manager cares about it? IMO it's more about transportation than stock.

**Comment by gurneyalex** [comment]
:+1:

**Pull request merged**